server/WebAPI/DTO/ActorDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPI.DTO
+{
+    public class ActorDTO
+    {
+        public string FullName { get; set; }
```

Comment:
don\t you think that other actors DTO could be neede in other places? it is too common name